docs/components/expansion_panel.md

```diff
@@ -0,0 +1,17 @@
+## Overview
+
+Accordion / Expansion panel and is based on the [Angular Material expansion panel component](https://material.angular.io/components/expansion/overview).
```

Comment:
I'd add a little more description here.

For example, this is a useful component for showing a summary header which can be expanded into a more detailed card/panel. 

Also, i'd explain the distinction between accordion and expansion panel here (even though you already cover it below). It's a little confusing because most people are going to think accordion (expansion panel is kind of an obscure name).

Reply:
Great point. I got a bit lazy on the docs. Haha.